magento/pwa-studio
# [PWA-428] Update upward-connector PWA install docs

## Description

Current https://magento.github.io/pwa-studio/tutorials/cloud-deploy/ docs is based on pre built dist. But we have removed dist from publishing. So, doc needs update on how to build part of magento cloud installation and upward.yml file path to use.

## Related Issue





* [[PWA-428](https://jira.corp.magento.com/browse/PWA-428)] Update upward-connector PWA install docs

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Follow instructions in article
2. Verify no errors during build and deployment
3. Verify Venia storefront actually works following these steps

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
